tests/testsrc/test262.properties

```diff
@@ -1128,17 +1517,9 @@ built-ins/JSON 45/165 (27.27%)
     stringify/value-bigint-tojson-receiver.js
     stringify/value-object-proxy.js
 
-built-ins/Map 12/185 (6.49%)
-    prototype/getOrInsert/append-new-values.js
-    prototype/getOrInsert/append-new-values-normalizes-zero-key.js
-    prototype/getOrInsert/append-value-if-key-is-not-present-different-key-types.js
-    prototype/getOrInsert/getOrInsert.js
-    prototype/getOrInsert/length.js
-    prototype/getOrInsert/name.js
-    prototype/getOrInsert/not-a-constructor.js
-    prototype/getOrInsert/returns-value-if-key-is-not-present-different-key-types.js
-    prototype/getOrInsert/returns-value-if-key-is-present-different-key-types.js
-    prototype/getOrInsert/returns-value-normalized-zero-key.js
+built-ins/Map 35/204 (17.16%)
+    prototype/getOrInsertComputed 19/19 (100.0%)
+    prototype/getOrInsert 14/14 (100.0%)
```

Comment:
How should this be interpreted?
Before: 10 tests in the `prototype/getOrInsert` were failing (and at least one passing)
Now: all tests in the `prototype/getOrInsert` are failing.
Is this the correct interpetion?